src/function.calls.spec.ts

```diff
@@ -155,4 +155,76 @@ describe("Function calls detector Agent Tests", () => {
     const findings: Finding[] = await handleTransaction(txEvent);
     expect(findings).toStrictEqual([findingGenerator({ to, from, input })]);
   });
+
+  it("Should be able to handle additional filter function passed inside functionCallGenerator", async () => {
+    const abiItem = {
+      name: "myMethod",
+      type: "function",
+      inputs: [
+        {
+          type: "uint256",
+          name: "myNumber",
+        },
+        {
+          type: "uint256",
+          name: "myNumber",
+        },
+      ],
+    } as AbiItem;
+
+    const filterFn = (value: TraceAction): boolean => {
+      const input = value.input;
+      const decode = abiDecode([abiItem], input) as any;
```

Comment:
Can you add a function to `utils.ts` with a more descriptive name and with only one purpose.

Something like `decodeFunctionInput` or `decodeFunctionCallParameters` should be fine.

Also notice that `decodeParameter(abiItem, input.slice(10))` will do the work. Then I think that we don't need to add more dependencies to the project ( i.e `abi-decoder` )

Reply:
There is already a function for that in utils, it is call `decodeParametersFromFunctionCall`